### ISSUE TITLE
Suggestion to fix events in tethered element

### ISSUE THREAD

**Issue by stevoland** [souporserious/react-tether#17]
If, for example you add a button to a tethered element, and click the button, the react event won't be handled because Tether has moved the node out of the React-managed hierarchy to `document.body`.

I think this can be fixed by:

 - Forget about creating `_elementParentNode`
 - `TetherComponent` renders and empty `<span/>`
 - `_elementNode = findDOMNode(this)`
 -  initialise Tether on `_elementNode`
 - `renderSubtreeIntoContainer` into `_elementNode`

Seems to work in my project where I've added a similar component.

I may have time for a PR if there's interest?

Cheers

**Comment by heavi5ide** [comment]
Events work fine for me -- I have a dropdown menu that uses react-tether for the menu that pops up, and the event handling works seamlessly. My component that contains the TetherComponent has a callback that is passed into the tethered element via a prop, and it gets called just fine when the event fires in the popup menu.

I've confirmed that the tethered component is being placed in the DOM directly in the body, outside of the root element of my React app. This is with React 0.14.8. It was also working with 0.14.6.

I tried briefly with React 15 over the weekend and nothing worked at all. When I have time I might try to isolate what exactly is going wrong; at the time I just noted that everything was broken and reverted to 0.14.8.
